package.json

```diff
@@ -5,8 +5,9 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "build": "./node_modules/grunt-cli/bin/grunt build",
-    "test": "./node_modules/grunt-cli/bin/grunt test"
+    "build": "grunt build",
```

Comment:
why did we change this? doesn't this mean grunt needs to be installed -g. Thought we did not like doing this?